### ISSUE TITLE
Starter Material UI Theme

### ISSUE THREAD

**Issue by amaliebarras** [CodeForBoulder/upswyng#30]
We need to come up with a material UI theme that we can start with, and then once Purdue is done with the branding and style guide we can update it.

**Comment by amaliebarras** [comment]
https://cimdalli.github.io/mui-theme-generator/

**Comment by amaliebarras** [comment]
Actually, let's just use the default to start? Then we can create a new ticket once Purdue updates style guidelines.
